src/lib.rs

```diff
@@ -0,0 +1,39 @@
+pub mod contract;
+pub mod error;
+pub mod field;
+pub mod transition;
+pub mod r#type;
+
+pub use contract::*;
+pub use error::Error;
+pub use field::*;
+pub use r#type::*;
+pub use transition::*;
+
+use std::{path::Path, process::Command};
+
+/// Run the scilla-fmt command using docker to generate a s-expression out of a given scilla contract.
+pub fn run_scilla_fmt(path: &Path) -> Result<String, Error> {
+    let volume = &format!(
+        "{}:/tmp/input.scilla",
+        path.canonicalize().unwrap().display()
+    );
+
+    let output = Command::new("docker")
+        .args([
+            "run",
+            "--rm",
+            "-v",
+            volume,
+            "-i",
+            "zilliqa/scilla:v0.13.3",
```

Comment:
Point of order - this should probably be 0.13.5 now .. (that being the latest version), and we should probably point up that this is how the trick is done in the README - not least because that means this crate will only work on x86_64 (since that's the only architecture we build the scilla container for)

Reply:
Thanks! In the upcoming commit, I'm going to use our rust implementation of the parser. So it'll address your concern.